Eniko/index.js

```diff
@@ -0,0 +1,76 @@
+// Check if two given integer values are in the range 50 to 99 (this includes both 50 and 99). Print true if either of them are in the range.
+let value = 101;
```

Comment:
Here you need 2 Integer numbers means 2 variables and print if either of them are in the range